clients/python/agentic-sandbox-client/k8s_agent_sandbox/async_k8s_helper.py

```
@@ -111,13 +113,26 @@ async def create_sandbox_claim(
+            try:
+                return await self.custom_objects_api.create_namespaced_custom_object(
+                    group=CLAIM_API_GROUP,
+                    version=CLAIM_API_VERSION,
+                    namespace=namespace,
+                    plural=CLAIM_PLURAL_NAME,
+                    body=manifest,
+                )
+            except ApiException as e:
+                if e.status == 429 and attempt < max_retries - 1:
+                    retry_after = float(e.headers.get("Retry-After", "1")) if e.headers else 1.0
+                    delay = max(retry_after, 0.5) + random.uniform(0.1, 0.5) * (attempt + 1)
```

Ruff (0.16.0)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)